### PR TITLE
[zh] Translate kubernetes-api/workload-resources/cron-job-v1 into Chi…

### PR DESCRIPTION
**Pull request by twilight0620**
Fixes: https://github.com/kubernetes/website/issues/32773

Do not have chinese docs: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/cron-job-v1/

so need to translate kubernetes-api/workload-resources/cron-job-v1.md into Chinese

### CODE REVIEW

**Comment by twilight0620** [comment]
The logs of failed checks are :Command failed with exit code 2: git submodule update --init --recursive --depth 1 && make deploy-preview . Anyone knows the reason?

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 1]
```diff
@@ -0,0 +1,1559 @@
...
+    从此模板创建的作业的标准对象元数据。
+    更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+    
+  <!--
+  - **jobTemplate.metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+  
+      Standard object's metadata of the jobs created from this template. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  -->
+
+  - **jobTemplate.spec**
```
Comment (jihoon-seo) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:132]: This line is different from the English page.

```suggestion
  - **jobTemplate.spec** (<a href="{{< ref "../workload-resources/job-v1#JobSpec" >}}">JobSpec</a>)
```

Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:132]: ok, i will delete this href temporarily

Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:132]: so i delete the ref, but now the checks also failed.

Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:132]: although i delete the href,  is it still has error?

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 2]
```diff
@@ -0,0 +1,1559 @@
...
+
+## CronJob {#CronJob}
+
+CronJob 代表单个定时作业的配置。
+
+<!--
+## CronJob {#CronJob}
+
+CronJob represents the configuration of a single cron job.
+-->
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:39]: 请按惯例，将英文注释放在中文翻译之前。下同。

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 3]
```diff
@@ -0,0 +1,1559 @@
...
+  
+<!--
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+-->  
+
+- **spec** (<a href="{{< ref "../workload-resources/cron-job-v1#CronJobSpec" >}}">CronJobSpec</a>)
+
+  指定 cron 作业所期望的行为，包含明细表。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:69]:
```suggestion
  指定 cron 作业所期望的行为，包含时间规划。
```

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 4]
```diff
@@ -0,0 +1,1559 @@
...
+    
+  <!--
+  - **jobTemplate.metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+  
+      Standard object's metadata of the jobs created from this template. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  -->
+
+  - **jobTemplate.spec**
+
+    作业所需行为的规范。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:134]:
```suggestion
    作业预期行为的规约。
```

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 5]
```diff
@@ -0,0 +1,1559 @@
...
+    
+  <!--
+   - **jobTemplate.spec** (<a href="{{< ref "../workload-resources/job-v1#JobSpec" >}}">JobSpec</a>)
+   
+       Specification of the desired behavior of the job. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+  -->
+
+- **schedule** (string)，必填
+
+  Cron 格式的时间表，请参阅：https://en.wikipedia.org/wiki/Cron.
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:145]:
```suggestion
  Cron 格式的时间表，请参阅：https://en.wikipedia.org/wiki/Cron。
```

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 6]
```diff
@@ -0,0 +1,1559 @@
...
+<!--
+- **schedule** (string), required
+
+  The schedule in Cron format, see https://en.wikipedia.org/wiki/Cron.
+-->  
+
+- **concurrencyPolicy** (string)
+
+  指定如何处理作业的并发执行。
+  有效值为： - "Allow"（默认）：允许 CronJobs 并发运行；- "Forbid"：禁止并发运行，如果上一次运行尚未完成则跳过下一次运行；- "Replace"：取消当前正在运行的作业并用新作业替换它。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:156]:
```suggestion
  有效值为：

    - "Allow"（默认）：允许 CronJobs 并发运行；
    - "Forbid"：禁止并发运行，如果上一次运行尚未完成则跳过下一次运行；
    - "Replace"：取消当前正在运行的作业并用新作业替换它。 
```

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 7]
```diff
@@ -0,0 +1,1559 @@
...
+    - **concurrencyPolicy** (string)
+
+     Specifies how to treat concurrent executions of a Job. Valid values are: - "Allow" (default): allows CronJobs to run concurrently; - "Forbid": forbids concurrent runs, skipping next run if previous run hasn't finished yet; - "Replace": cancels currently running job and replaces it with a new one
+   -->
+   
+  可能的枚举值:
+   - `"Allow"` 允许 CronJobs 并发运行。
+   - `"Forbid"` 禁止并发运行，如果上一次运行尚未完成则跳过下一次运行。
+   - `"Replace"` 取消当前正在运行的作业并用新作业替换它。
+
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:167]: 重复了

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 8]
```diff
@@ -0,0 +1,1559 @@
...
+<!--
+- **startingDeadlineSeconds** (int64)
+
+  Optional deadline in seconds for starting the job if it misses scheduled time for any reason.  Missed jobs executions will be counted as failed ones.
+-->
+
+- **suspend** (boolean)
+
+  此标志告诉控制器暂停后续执行，它不适用于已经开始的执行。默认为假。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:188]:
```suggestion
  此标志告诉控制器暂停后续执行，它不适用于已经开始的执行。默认为 false。
```

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 9]
```diff
@@ -0,0 +1,1559 @@
...
+- **active** ([]<a href="{{< ref "../common-definitions/object-reference#ObjectReference" >}}">ObjectReference</a>)
+
+  *Atomic: will be replaced during a merge*
+  
+  A list of pointers to currently running jobs.
+-->
+
+- **lastScheduleTime** (Time)
+
+  上次成功安排作业的时间信息。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:249]:
```suggestion
  上次成功调度作业的时间信息。
```

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 10]
```diff
@@ -0,0 +1,1559 @@
...
+#### Parameters
+-->
+
+
+- **namespace** (*in path*): string，必填
+
+  <a href="{{< ref "../common-parameters/common-parameters#namespace" >}}">namespace</a>
+
+
+- **body**: <a href="{{< ref "../workload-resources/cron-job-v1#CronJob" >}}">CronJob</a>, required
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:773]: required 请统一翻译为“必需”，保持一致

```suggestion
- **body**: <a href="{{< ref "../workload-resources/cron-job-v1#CronJob" >}}">CronJob</a>，必需
```

Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md]: I have modified the comments. Thanks for your review.

**Comment by twilight0620** [comment]
The pages in workload-resources quote each other...every pages quote other pages in wordload-resource, and none of pages in workload-resources exists...the check will always failed...

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 11]
```diff
@@ -0,0 +1,1545 @@
...
+   <a name="JobTemplateSpec"></a>
+   *JobTemplateSpec describes the data a Job should have when created from a template*
+-->
+
+- **jobTemplate** (JobTemplateSpec)，必需
+
+  指定执行 CronJob 时将创建的作业。
+  
+  <a name="JobTemplateSpec"></a>
+  *JobTemplateSpec 描述了从模板创建作业时应具有的数据。*
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:115]:
```suggestion
  **JobTemplateSpec 描述了从模板创建作业时应具有的数据。**
```

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 12]
```diff
@@ -0,0 +1,1545 @@
...
+- **active** ([]<a href="{{< ref "../common-definitions/object-reference#ObjectReference" >}}">ObjectReference</a>)
+
+  *Atomic: will be replaced during a merge*
+  
+  A list of pointers to currently running jobs.
+-->
+
+- **active** ([]<a href="{{< ref "../common-definitions/object-reference#ObjectReference" >}}">ObjectReference</a>)
+
+  *Atomic: 将在合并期间被替换。*
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:233]:
```suggestion
  **原子性字段：将在合并期间被替换。**
```

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 13]
```diff
@@ -0,0 +1,1545 @@
...
+  <a name="Time"></a>
+  *Time is a wrapper around time.Time which supports correct marshaling to YAML and JSON.  Wrappers are provided for many of the factory methods that the time package offers.*
+-->
+
+- **lastScheduleTime** (Time)
+
+  上次成功调度作业的时间信息。
+  
+  <a name="Time"></a>
+  *Time 是 time.Time 的包装器，它支持 YAML 和 JSON 格式。time 包提供的许多工厂方法都提供了包装器。*
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:251]:
```suggestion
  **Time 是 time.Time 的包装器，它支持 YAML 和 JSON 格式。time 包提供的许多工厂方法都提供了包装器。**
```

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 14]
```diff
@@ -0,0 +1,1545 @@
...
+  *Time is a wrapper around time.Time which supports correct marshaling to YAML and JSON.  Wrappers are provided for many of the factory methods that the time package offers.*
+-->  
+
+
+- **lastSuccessfulTime** (Time)
+
+  上次成功完成作业的时间信息。
+
+  <a name="Time"></a>
+  *Time 是 time.Time 的包装器，它支持 YAML 和 JSON 格式。time 包提供的许多工厂方法都提供了包装器。*
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:268]:
```suggestion
  **Time 是 time.Time 的包装器，它支持 YAML 和 JSON 格式。time 包提供的许多工厂方法都提供了包装器。**
```

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md (hunk 11 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:115]: 这块英文就是一个星的

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md (hunk 12 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:233]: 这些英文的都是一个星的 包括下面的，中文需要改么？我看一个星的字体和英文字体是一样的，2个星就不一样了

Comment (tengqm) [reply content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:233]: 一个星是斜体，对英文可接受，对中文很难看。所以遇到一个星的情况，可以考虑改成两个星（粗体）。

Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md]: 好的 已修改。

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 15]
```diff
@@ -0,0 +1,1545 @@
...
+
+<!--
+## Operations {#Operations}
+
+
+
+<hr>
+-->
+
+## Operations {#Operations}
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:326]:
```suggestion
## 操作 {#Operations}
```

**Comment by tengqm** [comment]
Please squash your PR into a single commit.

**Comment by tengqm** [comment]
The netlify is complaining

```
9:28:57 AM: ERROR 2022/05/11 01:28:57 [zh] REF_NOT_FOUND: Ref "../workload-resources/job-v1": "/opt/build/repo/content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:129:37": page not found
```

**Comment by twilight0620** [comment]
> job-v1

job-v1 page does not exist. And job-v1 page refer to pod-template-v1 page,which also does not exist. And pod-template-v1 page refer to pod-v1 page which does not exist. And pod-v1 also refer to other pages in workload-resources...so  i do not how to resolve it..

**Comment by tengqm** [comment]
@twilight0620 There is no circular references in the set. So we'd better start with some basic concepts.

File: content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 16]
```diff
@@ -0,0 +1,1545 @@
...
+      Standard object's metadata of the jobs created from this template. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  -->
+
+  - **jobTemplate.metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+    从此模板创建的作业的标准对象元数据。
+    更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+    
+  <!--
+   - **jobTemplate.spec** (<a href="{{< ref "../workload-resources/job-v1#JobSpec" >}}">JobSpec</a>)
```
Comment (Sea-n) [inline content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:129]: The target file is not existed yet, it caused build failure.
```suggestion
   - **jobTemplate.spec** (<a href="{ {<-- ref "../workload-resources/job-v1#JobSpec" >} }">JobSpec</a>)
```

**Review by Sea-n — COMMENTED** [review]
Please refer to [Netlify build log](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/627b6c9ec208750008bdb443#L1287), feel free to `/unhold` yourself once the issue is solved.

/hold

**Comment by Sea-n** [comment]
Hey @twilight0620, we have moved Mandarin contain from `content/zh/docs` to `content/zh-cn/docs`, please rebase and move you file accordingly.

**Comment by twilight0620** [comment]
> Hey @twilight0620, we have moved Mandarin contain from `content/zh/docs` to `content/zh-cn/docs`, please rebase and move you file accordingly.

ok

**Comment by twilight0620** [comment]
ping @cici37

**Pull request closed without merging**